test/codegen.jl

```diff
@@ -466,6 +466,9 @@ end
     end
 end
 
+@testset "safe_fn" begin
+    name = "julia_^
+    @test safe_fn(name) != name 
```

Comment:
Probably needs `CUDAnative.safe_fn`?
Which kernel/function did this trigger with? Maybe that would be an interesting test too.

Reply:
This was triggered by the `^` function (acting on intervals). I'm not sure how to test that?

Reply:
The function name was `julia_^_63453`

Reply:
```
julia> @eval @noinline $(Symbol("dummy_^"))(x) = x
dummy_^ (generic function with 1 method)

julia> @eval kernel_341(a,i) = (@inbounds a[1]=$(Symbol("dummy_^"))(a[1]); nothing)

julia> CUDAnative.code_sass(kernel_341, Tuple{CuDeviceArray{Int,2,AS.Global},Int})
ptxas application ptx input, line 19; fatal   : Parsing error near '"julia_dummy_^_12806"': syntax error
```